doc/setting_up_sidewalk_environment/setting_up_sdk.rst

```diff
@@ -61,16 +61,7 @@ Follow these steps to download the Sidewalk application for the nRF Connect SDK:
        |___ zephyr
        |___ ...
 
-#. Check out the branch that contains the west manifest file with the Sidewalk group filter:
-
-   .. code-block:: console
-
-      $ cd nrf
-      $ git checkout main
-      $ git pull
-      $ cd ..
-
-#. Set the Sidewalk application manifest and update.
+#. Check manifest file. This step is necessary if you worked with Sidewalk as application with manifest repository before.
```

Comment:
```suggestion
#. If you have worked with the Sidewalk application in a previous distribution model (as a manifest repository), check the manifest file.
   Make sure the manifest path points to :file:`west.yml` inside the :file:`nrf` directory:
```
Not sure if the meaning is conveyed - please let me know if something doesn't seem right